### ISSUE TITLE
no request for youtube credentials

### ISSUE THREAD

**Issue by shelmz** [kihapper/TheirTube#2]
there is no request for youtube credentials when starting the script and therefore there appears no _/cookies/theirtube_cookie_
only the daily scraping time is displayed.

**Comment by kihapper** [comment]
Yes, so you need to log in manually within the puppeteer browser when you activate it for the first time to create a browser cookie with a logged-in state.

You can also login automatically by adding  
_theirtube.login(username, password)_ within the index.js file.

**Comment by jtisaacs** [comment]
@kihapper If I want to automatically login to multiple accounts how would I add multiple usernames and passwords. I see the variable persona_list on line 25 of index.js. Does persona1, persona2, persona3, ... represent different usernames? If so, is it possible to add a password for each of those usernames.

I assume that I need to add the line:
theirtube.login(persona, password) 
on line 54 of index.js.

**Comment by havardl** [comment]
@kihapper bump. Is it possible to do what @jtisaacs is describing above?
